R/deploy.R

```diff
@@ -448,89 +456,157 @@ get_image <- function(content, path = NULL) {
   return(fs::as_fs_path(path))
 }
 
-#' @rdname get_image
+#' @rdname get_thumbnail
 #' @export
-delete_image <- function(content, path = NULL) {
-  warn_experimental("delete_image")
+get_image <- function(content, path = NULL) {
+  lifecycle::deprecate_warn("0.3.1", "get_image()", "get_thumbnail()")
+  get_thumbnail(content, path)
+}
+
+#' @rdname get_thumbnail
+#' @export
+delete_thumbnail <- function(content, path = NULL) {
   validate_R6_class(content, "Content")
   guid <- content$get_content()$guid
 
   con <- content$get_connect()
 
   if (!is.null(path)) {
     scoped_experimental_silence()
-    get_image(content, path)
+    get_thumbnail(content, path)
```

Comment:
Maybe the function names are just confusing to me, but why does `delete_thumbnail()` have to `get` first? Why is there a `path` argument to delete?

Reply:
The way it has existed previously, `delete_thumbnail()` saves the image to `path` if `path` is provided before deleting. I've tried to scope the changes in this PR to renaming / deprecating, not also changing the functionality.

Reply:
Another option would be to keep the old implementation as `delete_image()` and have the new `delete_thumbnail()` _only_ delete, not also get.

Reply:
> Another option would be to keep the old implementation as the deprecated `delete_image()` and have the new `delete_thumbnail()` _only_ delete, not also get.

Yeah I like that 🚮

Reply:
Like, users can `GET` before they `DELETE` if they want, we're only saving people one line of code in an uncommon edge case.

Reply:
💯